examples/grafana/README.md

```diff
@@ -0,0 +1,27 @@
+# grafana-ocp
+
+This template creates a custom Grafana instance preconfigured to gather Prometheus openshift metrics.
+It is uses OAuth token to login openshift Prometheus.
+
+
+## To deploy grafana
+Note: make sure to have openshift prometheus deployed.
+(https://github.com/openshift/origin/tree/master/examples/prometheus)
+
+1. ```oc create namespace grafana```
+2. ```oc new-app -f grafana-ocp.yaml```
+3. grab the grafana url ``` oc get route |awk 'NR==2 {print $2}' ```
+4. grab the ocp token, from openshift master run: ```oc sa get-token prometheus -n kube-system```
```

Comment:
@smarterclayton hi, do you have preference on how to get a token for the Prometheus auth proxy ?

Here we assume the namespace is `kube-system`, but if we install Prometheus using openshift-ansible [1] the namespace will be `prometheus`.

[1] https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_prometheus/defaults/main.yaml#L5

cc @zgalor @cben @joelddiaz

Reply:
@yaacov 
I've start to work on another patch, to automate the deployment process.

I use this command,
`oc get route --all-namespaces |grep -i prometheus |awk '{print $3}'`


let's merge this patch for initial stage, then lets work on other things like https, automation and so.
@smarterclayton